source/includes/events/_create.json.md

```diff
@@ -0,0 +1,53 @@
+```json
+{
+   "event":{
+      "_type":"event",
+      "id":1058592,
+      "created_at":"2018-11-02T10:44:27.000+00:00",
+      "updated_at":"2018-11-02T10:44:27.000+00:00",
+      "title":"Awesome Conf",
+      "description":null,
+      "banner":{
+         "url":null,
+         "thumb":{
+            "url":null
+         }
+      },
```

Comment:
I couldn't actually see anywhere in Dashboard that we were using this? Is it worth this still being in the response?